docs/administration/robot-accounts/_index.md

```diff
@@ -0,0 +1,141 @@
+
+   ![Copy system robot account token](../../img/copy-system-robot-account-token.png)
+
+   {{< important >}}
+   Harbor does not store robot account secrets, so you must either download the secret or copy and paste its contents into a text file. There is no way to get the secret from Harbor after you have created the robot account.
+   {{< /important >}}
+
+   The new robot account appears as `<prefix>account_name` in the list of robot accounts. Read more about [robot account prefixes](#configure-robot-account-prefix).
+
+## Edit, Disable, or Delete a System Robot Account
+
+You are able to edit, disable, or delete a sytem robot account.
+
+1. From the administrator **Robot Account** page, select the checkbox next to the robot account you are updating.
+1. Select **Action** and then **Edit**, **Disable**, or **Delete**.
+
+  ![Disable or delete a robot account](../../img/disable-delete-system-robot-account.png)
+
+## Refresh System Robot Account Secret
+
+You can refresh a robot account's secret after its created in the event that you need a new one.
+
+1. From the administrator **Robot Account** page, select the checkbox next to the robot account you are updating.
+1. Select **Action** and then **Refresh Secret**.
+1. By default Harbor will generate a new secret randomly, or you can choose to enable manually reseting the secret and entering the **New Secret** then **Confirm Secret**.
+
+    ![Refresh system robot account secret](../../img/refresh-robot-account-token.png)
+
+1. Click **Refresh**. If you created a secret randomly, download the secret JSON file or copy and paste its contents.
+
+## Configure the Expiry Period of Robot Accounts
+
+By default, robot accounts expire after 30 days. You can set a longer or shorter lifespan for robot accounts by modifying the expiry period for robot account tokens. The expiry period applies to all robot accounts in all projects.
+
+1. Log in to the Harbor interface with an account that has Harbor system administrator privileges.
+1. Go to **Configuration** and select **System Settings**.
+1. In the **Robot Token Expiration (Days)** row, modify the number of days after which robot account tokens expire.
+
+    ![Set robot account token expiry](../../img/set-robot-account-token-duration.png)
+
+## Configure Robot Account Prefix
+
+By default, robot account names use a prefix of `robot$`. Harbor uses this prefix to distinguish a robot account from a user account. The full name of a robot account is the prefix and the name you provided when creating the robot account. For example if you create a new robot account with the name `test`, the full name is  `robot$test`.
+
+The same prefix is used for all robot accounts, including both system and project robot accounts. When you update this value, it will apply to all existing and future system and project robot accounts, including robot accounts created in Harbor v2.1 and earlier.
```

Comment:
The prefix will not be updated for the robots created at v2.1 or before. The prefix editing only works for v2.2+ robots.

Reply:
Thank you, i updated this information.